CMakeLists.txt

```
@@ -354,6 +375,34 @@ if(GUI)
             ${THREADING_LINK_LIBS}
   )
 
+  # Main Dissolve GUI
+  target_link_libraries(
+    ${qmlgui_target_name}
+    PUBLIC ${WHOLE_ARCHIVE_FLAG}
+            ${BASIC_LINK_LIBS}
+            # Module gui libs
+            ${MODULEGUI_LINK_LIBS}
+            moduleWidgetProducer
+            # Main gui libs
+            render
+            widgets
+            keywordWidgets
+            delegates
+            models
+            gui
+            ${NO_WHOLE_ARCHIVE_FLAG}
+    PRIVATE # External libs
+            Qt6::Widgets
+            Qt6::Core
+            Qt6::Qml
+            OpenGL::GL
+            ${GUI_LINK_LIBS}
+            ${FREETYPE_LIBRARIES}
+            ${FTGL_LIBRARIES}
+            ${CORE_LINK_LIBS}
+            ${THREADING_LINK_LIBS}
```

```suggestion
            Qt6::Qml
            ${CORE_LINK_LIBS}
            ${THREADING_LINK_LIBS}
```

Cutting down dependencies again